drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1416 @@
+[note]: <> ("note to self and file editor, there are specific comments left throughout this file, for the sake of understanding the author's reason for using a particular method of approach; please feel free to delete once acknowledged or reply edit with a better approach 🥂")
+
+# How to Build a DAO on the Fuel blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered an essential blockchain application.
```

Comment:
Please make sure to always capitalize DAO throughout the article. I've seen a few places where it's written "dao", a quick search and replace should fix that! :)

Reply:
Done 
Thanks for clearing this out.
I've a;so changed all "dao" to "DAO"